extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/dev/HibernateOrmDevServicesProcessor.java

```diff
@@ -44,7 +44,7 @@ void devServicesAutoGenerateByDefault(List<JdbcDataSourceSchemaReadyBuildItem> s
 
             if (!managedSources.contains(dataSourceName.orElse(DataSourceUtil.DEFAULT_DATASOURCE_NAME))) {
                 String databaseGenerationPropertyKey = HibernateOrmRuntimeConfig.puPropertyKey(entry.getKey(),
-                        "database.generation");
+                        "schema-management.strategy");
                 if (!ConfigUtils.isAnyPropertyPresent(propertyKeysIndicatingDataSourceConfigured)
                         && !ConfigUtils.isPropertyNonEmpty(databaseGenerationPropertyKey)) {
```

Comment:
You probably want to check `ConfigUtils.isPropertyNonEmpty` for _both_ property keys (old one and new one), no?

Reply:
You're totally right, thanks for catching it. Will push an update soon.

Reply:
Done!